lib/npm.js

```diff
@@ -30,11 +78,23 @@ export let mergeManifests = (existing, generated) => {
 	return result
 }
 
+/**
+ * Create the component's npm name from its bower name
+ * @param {string} componentName the name of the component
+ * @param {string} npmOrganisation the organisation to use (default: `financial-times`)
```

Comment:
Shouldn't stop merging, but we could use an optional param with a default here. Thoughts?

```suggestion
 * @param {string} [npmOrganisation="financial-times"] the organisation to use
```